modules/nf-core/tcoffee/align/main.nf

```diff
@@ -39,14 +39,6 @@ process TCOFFEE_ALIGN {
         -outfile $outfile \
         $write_output
 
-    # If stdout file exist and compress is true, then compress the file
-    # This is a patch for the current behaviour of the regressive algorithm
-    # that does not support the stdout redirection
-    if [ -f stdout ] && [ "$compress" = true ]; then
-        pigz -cp ${task.cpus} < stdout > ${prefix}.aln.gz
-        rm stdout
-    fi
-
```

Comment:
was it intended to remove this?

Reply:
yes :) i moved the regressive (the one needing this lines) to a new module